src/__tests__/add-remove.test.js

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable */
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+
+const dom = new JSDOM(`<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>dom.window.document</title>
+</head>
+<body>
+<div id="app-container">
+<div id="app-header">
+    <h4>Today's To Do</h4>
+    <span id="reset-btn">⟳</span>
+</div>
+<form id="form-section" action="submit">
+    <input type="text" placeholder="Add to your list..." id="add-input">
+    <button id="form-btn" type="submit">⏎</button>
+</form>
+<ul id="task-list"></ul>
+<button type="button" id="clear-all">Clear all completed</button>
+</div>
+</body>
+</html>`);
+
+const localStorageMock = (() => {
+  let store = {};
+  return {
+    getItem(key) {
+      return store[key];
+    },
+    setItem(key, value) {
+      store[key] = value.toString();
+    },
+    clear() {
+      store = {};
+    },
+    removeItem(key) {
+      delete store[key];
+    },
+  };
+})();
+Object.defineProperty(dom.window, 'localStorage', {
+  value: localStorageMock,
+});
+
+const taskArr = [
+  {
+    description: 'Task 1',
+    completed: false,
+    taskId: 1,
+  },
+];
+localStorageMock.setItem('taskArr', JSON.stringify(taskArr));
+
+function removeTask(iD) {
+  const taskArr = JSON.parse(localStorageMock.getItem('taskArr')) || [];
+
+  taskArr.forEach((task, index) => {
+    if (Number(task.taskId) === Number(iD)) {
+      taskArr.splice(index, 1);
+    }
+  });
+  localStorageMock.setItem('taskArr', JSON.stringify(taskArr));
+}
+
+function addTasktoList(task) {
+  const taskCtn = dom.window.document.getElementById('task-list');
+  const taskLi = dom.window.document.createElement('li');
+  // taskLi.setAttribute('id', task.taskId);
```

Comment:
- [ ] Please remove this commented code because it is a bad practice comments should be reserved to explain difficult-to-understand code or the reason for doing something out of the ordinary. Don't worry code should be stored in source control :+1: